### DISCUSSION TITLE
SRT protocol support

### DISCUSSION THREAD

**Discussion by alexp2021** [pedroSG94/RootEncoder#1257] [Q&A]
Hello Pedro,

Are you going to implement streaming via SRT protocol?
As you answered in #979, the main reason of not supporting is absence of good free server for testing.
As for now SRS support SRT.
Do you have plans to implement SRT support?

**Comment by pedroSG94** [comment]
Hello,
It is good news. I will check it but I can't promise anything.
HLS is also under consideration
